.github/actions/dead-links/action.yml

```diff
@@ -0,0 +1,17 @@
+name: Dead Links
+description: Run Dead Links Tests
+runs:
+  using: "composite"
+  steps: 
+    - run: npm ci
+      shell: bash
+    - run: npm run doc-prepare
+      shell: bash
+    - run: $(npm bin)/kuzdoc iterate-repos:install --repos_path doc/framework/.repos/
```

Comment:
As far as I understand what happens here, it test only the proper JS SDk docs `run: cd doc/framework/ && HYDRA_MAX_CONCURRENCY=20 ruby .ci/dead-links.rb -p src/sdk/js/7/` but it may need to access docs inside other repo right ?

So for me this doesn't need to change